aim/sdk/adapters/sb3.py

```diff
@@ -0,0 +1,136 @@
+import logging
+import numpy as np
+from typing import Optional, Dict, Any, Union, Tuple
+
+from stable_baselines3.common.logger import KVWriter, Logger
+from stable_baselines3.common.callbacks import BaseCallback  # type: ignore
+
+from aim.sdk.run import Run
+from aim.ext.resource.configs import DEFAULT_SYSTEM_TRACKING_INT
+
+
+logger = logging.getLogger(__name__)
+
+
+class AimOutputFormat(KVWriter):
+    """
+    Track key/value pairs into Aim run.
+    """
+
+    def __init__(
+        self,
+        aim_callback
+    ):
+        self.aim_callback = aim_callback
+
+    def write(
+        self,
+        key_values: Dict[str, Any],
+        key_excluded: Dict[str, Union[str, Tuple[str, ...]]],
+        step: int = 0,
+    ) -> None:
+        for (key, value), (_, excluded) in zip(
```

Comment:
lets check the existence/value of the `key` in `key_excluded` dict. it will allow to handle cases when `key_excluded` contains only the sub-set of actually excluded keys.

Reply:
@alberttorosyan great idea.
Though just double-checked, those two dicts always have the same length and contain the same keys.
Here is an example of the keys and values of two dicts
- `key_values = {'train/learning_rate': 0.0007, 'train/n_updates': 99,...} `
- `key_excluded = {'train/learning_rate': None, 'train/n_updates': tensorboard,...} `

The keys that have a None value will be tracked.